### PR TITLE
Automatically assembles items list

### LINKED ISSUE #19: Remove the items list from benchmarker.py
Instead iterate through the keys of the main JSON file for each language.
Construct the list as you go along.

### PR DESCRIPTION
**Pull request by naslundx**
Fixes #19 and we can now use `--include=` and `--exclude=` flags.

@dekespo Does this make sense to you? :)

### CODE REVIEW

**Comment by naslundx** [comment]
Now it's possible to do really nice stuff like

`python3 benchmarker.py --iterations=1 --exclude=go,binarytree,densematrix,ooc`

or

`python3 benchmarker.py --include=c,cpp,python3,helloworld,primes`

or a combination of the two, like the cool

`python3 benchmarker.py --include=c,cpp,go --exclude=binarytree --iterations=10`

which runs everything except binarytree on c, c++ and go 10 times.

File: benchmarker.py [hunk 1]
```diff
@@ -5,14 +5,66 @@
...
 
 FNULL = open(os.devnull, 'w')
-languages = ["c", "cpp", "python2", "python3", "ooc", "rust", "go", "java", "erlang", "haskell", "js", "csharp", "lua", "ruby", "perl"] 
-items = ["helloworld", "primes", "densematrix", "binarytree"]  # TODO read from config file
+
+
+# Find all languages being used
+def get_languages(include_str="", exclude_str=""):
+	languages = []
+	possible_languages = [x[0] for x in os.walk(os.getcwd()) if not '.' in x[0] or '_' in x[0]]
```
Comment (marcusnaslund) [inline benchmarker.py]: missing a `not` in the second if clause

**Comment by dekespo** [comment]
Yes, it does and it's a great work!

**Comment by naslundx** [comment]
Thanks!

**Pull request merged**